api/lib/controllers/meeting.js

```diff
@@ -199,6 +199,72 @@ module.exports = {
     const participants = await Participant.find({ meeting_id: _id });
 
     return res.status( 200 ).send( participants );
+  },
+
+  async getAllMeetings( req, res ) {
```

Comment:
I think we should call it `index`instead of `getAllMeetings` just because it's the REST convention.

Reply:
@zbarnz can make this change as well?